ipatests/test_integration/test_installation.py

```diff
@@ -170,6 +170,37 @@ def test_replica2_with_ca_kra_install(self):
     def test_replica2_ipa_dns_install(self):
         super(TestInstallWithCA1, self).test_replica2_ipa_dns_install()
 
+    def test_install_with_bad_ldap_conf(self):
+        """
+        Test an client install with a non standard ldap.config
+        https://pagure.io/freeipa/issue/7418
+        """
+        ldap_conf = paths.OPENLDAP_LDAP_CONF
+        base_dn = self.master.domain.basedn  # pylint: disable=no-member
+        client = self.replicas[0]
+        tasks.uninstall_master(self.master)
+        tasks.uninstall_master(client)
+        tasks.install_master(self.master, setup_dns=False)
```

Comment:
Uninstalling master at line 181 and then installing here?

May be you only want to uninstall client (i.e replica from previous test) for this test.

Reply:
yes